app/jobs/regular/chat_notify_watching.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 module Jobs
+  # TODO (martin) A spec for this whole class.
```

Comment:
Note it is tested somewhat through `ChatMessageCreator`. It does appear that previously testing via integration tests was preferred, but I think any class like this should have tests of its own.

Reply:
Yes I fully agree, there's been multiple debates on this recently, but my position is:
- fully unit tested class
- acceptance test for important categories of events, eg in a controller: we test what happens when an error is raised in the underlying lib, not each kind of error message

Well this is my position at least :p

Reply:
Same :heart: